api/WebGLObject.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebGLObject",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": false
```

Comment:
https://developer.mozilla.org/docs/Web/API/WebGLObject is 404, do you know what this is supposed to be? It'll be false for everything, seems like it should be removed from the data?

Reply:
It looks like this API file was added by #1410, generated via https://github.com/dontcallmedom/webidl2mdnbcd from https://www.khronos.org/registry/webgl/specs/latest/1.0/.

Reply:
Alright, it's a base class, but at least Chromium doesn't use it. `false` here is OK then, maybe later someone will come and purge files that have to true or string values.